components/src/dynamo/vllm/tests/test_vllm_handlers_lora.py

```
@@ -0,0 +1,297 @@
+    pytest.mark.pre_merge,
+]
+
+
+class _StubHandler(BaseWorkerHandler):
+    """Concrete subclass so the ABC can be instantiated without __init__."""
+
+    def generate(self, request, context):  # pragma: no cover - never called
+        raise NotImplementedError
+
+
+def _make_handler(endpoint=object()) -> BaseWorkerHandler:
```

Ruff (0.15.20)

[warning] Do not perform function call `object` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)